Sources/UInt128.swift

```diff
@@ -662,20 +667,11 @@ extension UInt128 : ExpressibleByStringLiteral {
     // MARK: Type Methods
 
     internal static func _valueFromString(_ value: String) -> UInt128? {
-        let radix = UInt128._determineRadixFromString(value)
+        let radix = value.radix
```

Comment:
I’m having a hard time understanding what value this refactoring brings. Could you explain the rationale behind replacing a local method with a private extension?

Reply:
because radix is fundamentally a property of the string.

Reply:
I see where you're coming from, but I believe I'd like to keep the structure as it was.